src/components/alert/alert-component.ts

```diff
@@ -303,6 +342,10 @@ export class AlertCmp {
   }
 
   getValues(): any {
+    if (this.isCustomAlert) {
+      return this._viewCtrl.instance;
```

Comment:
Yes. Maybe there could be a transform function in the future to return relevant data only...